File: for_python/examples.py
from animedev import client, exceptions

def search_anime(anime_name):
    try:
        anime = client.search(anime_name)
        text = f'''
Anime Title: {anime['AnimeTitle']}
Anime Link: {anime['AnimeLink']}
Anime Image: {anime['AnimeImg']}
Search Query: {anime['Search_Query']}
'''

        print(text)
    except exceptions.NotFound as e:
        print(e)
        return
    except Exception as e:
        print(e)
        return

# Searching anime.
search_anime('Doraemon') # Add your anime name instead of Doraemon.

# Printing version of animedev.
import animedev
print(animedev.__version__)

File: for_node/examples.js
const animedev = require('./animedev')

const AnimeDevClient = new animedev.client()

try {
    // Searching anime.
    const anime = AnimeDevClient.search('Doraemon') // enter your anime name instead of Doraemon.
    console.log(anime)
} catch (err) {
    console.log(err)
}

// Printing version of animedev.
console.log(animedev.version)